code/game/machinery/Sleeper.dm

```diff
@@ -449,25 +453,23 @@
 /obj/machinery/sleeper/ex_act(severity)
 	if(filtering)
 		toggle_filter()
+	var/mob/M = src.occupant

```

Comment:
that var is to target the person inside the pod after they're ejected, If I just eject and qdel then the mob inside will not have its ex_act() called